pkg/controller/worker/machines.go

```diff
@@ -186,6 +187,11 @@ func (w *workerDelegate) generateMachineConfig() error {
 				MachineConfiguration: genericworkeractuator.ReadMachineConfiguration(pool),
 			})
 
+			//TODO : remove when AWS CSI driver stops using the aws csi topology key
+
+			//adding aws csi driver topology label if its not specified
```

Comment:
```suggestion
			// TODO: remove when AWS CSI driver stops using the aws csi topology key - <link-to-issue>
			// add aws csi driver topology label if its not specified
```
Can you please link the open issue regarding this on `aws-csi-driver` in this TODO comment?
Also please keep spaced-comments.